shared/js/l10n.js

```diff
@@ -1492,6 +1511,7 @@
         return getDirection(navigator.mozL10n.ctx.supportedLocales[0]);
       }
     },
+    gaiaVersion: '2.2',
```

Comment:
In https://bugzilla.mozilla.org/show_bug.cgi?id=1115797 and https://bugzilla.mozilla.org/show_bug.cgi?id=1115798 I was able to work around having to hardcode the version of Gaia by making use of the `deviceinfo.os` setting. That worked well because in both cases we're in the Settings app which has the right privileges to access settings (as the name would imply :)).

I'm not sure how to work around this here, though.  I don't think we'd want to add a dependency on mozSetting in l10n.js (in fact, we worked hard to remove it in the past).

I currently see two solution:

1. On buildtime, put the value of the `deviceinfo.os` setting in a `meta` element in each l10n-enabled HTML file.

2. When `getAdditionalLanguages` returns, save the most recent version number for each locale and use it for language negotation _and_ resource fetching.  This is based on the assumption that a user running 2.2 will be only taken to the Marketplace listing of langpacks for 2.2, so the versions will match initially.  When the user updates Gaia to 2.3 however, they will also need to update the langpack, or the translations will be outdated.  Maybe there's a way to push new langpacks for 2.3 on system update?  In any case, this approach avoids the problem of versioning for now.

@zbraniecki, what are your thoughts on this?